Probe/src/probe/probe.go

```diff
@@ -62,3 +62,14 @@ func (p *probe) probe(pwg *sync.WaitGroup) {
 	}
 	pwg.Done()
 }
+
+func (p *probe) getTypeString() string {
```

Comment:
If this is a proto enum, then you can say .String() and get the string that way

Reply:
Addressed in c9ccc1a